src/pages/resultsView/expression/expressionHelpers.ts

```diff
@@ -0,0 +1,212 @@
+        fill: "#A4A4A4",
+        stroke: "#B40404",
+        legendText: "Splice"
+    },
+    in_frame: {
+        typeName: "In_frame",
+        symbol: VictoryShapeType.square,
+        fill: "#DF7401",
+        stroke: "#B40404",
+        legendText: "In-frame"
+    },
+    nonstart: {
+        typeName: "Nonstart",
+        symbol: VictoryShapeType.plus,
+        fill: "#DF7401",
+        stroke: "#B40404",
+        legendText: "Nonstart"
+    },
+    nonstop: {
+        typeName: "Nonstop",
+        symbol: VictoryShapeType.triangleUp,
+        fill: "#1C1C1C",
+        stroke: "#B40404",
+        legendText: "Nonstop"
+    },
+    missense: {
+        typeName: "Missense",
+        symbol: VictoryShapeType.circle,
+        fill: "#DF7401",
+        stroke: "#B40404",
+        legendText: "Missense"
+    },
+    other: {
+        typeName: "Other",
+        symbol: VictoryShapeType.square,
+        fill: "#1C1C1C",
+        stroke: "#B40404",
+        legendText: "Other"
+    },
+    one_mut: {
+        typeName: "one_mut",
+        symbol: VictoryShapeType.circle,
+        fill: "#DBA901",
+        stroke: "#886A08",
+        legendText: "One Gene mutated"
+    },
+    both_mut: {
+        typeName: "both_mut",
+        symbol: VictoryShapeType.circle,
+        fill: "#FF0000",
+        stroke: "#B40404",
+        legendText: "Both mutated"
+    },
+    non_mut: {
+        typeName: "non_mut",
+        symbol: VictoryShapeType.circle,
+        fill: "#00AAF8",
+        stroke: "#0089C6",
+        legendText: "No Mutation"
+    },
+    non_sequenced: {
+        typeName: "non_sequenced",
+        symbol: VictoryShapeType.circle,
+        fill: "white",
+        stroke: "gray",
+        legendText: "Not sequenced"
+    }
+
+};
+
+export function getExpressionStyle(mutationType: string){
+    const mappedType = mutationVocabularyMap[mutationType] || 'other';
+    return ExpressionStyleSheet[mappedType] || ExpressionStyleSheet.other;
+}
+
+
+export const mutationVocabularyMap: { [key:string] : string } = {
+    "5'flank": "other",
+    "complex_indel": "other",
+    "essential_splice_site": "splice",
+    "exon skipping": "other",
+    "exon14skip": "other",
+    "frame_shift_del": "frameshift",
+    "frame_shift_ins": "frameshift",
+    "frameshift": "frameshift",
+    "frameshift deletion": "frameshift",
+    "frameshift insertion": "frameshift",
+    "frameshift_coding": "frameshift",
+    "frameshift_insertion": "frameshift",
+    "fusion": "other",
+    "in_frame_del": "in_frame",
+    "in_frame_ins": "in_frame",
+    "missense": "missense",
+    "missense_mutation": "missense",
+    "nonsense": "nonsense",
+    "nonsense_mutation": "nonsense",
+    "nonstop_mutation": "nonstop",
+    "splice": "splice",
+    "splice_site": "splice",
+    "splice_site_snp": "splice",
+    "splicing": "splice",
+    "translation_start_site": "nonstart",
+    "viii deletion": "other"
+}
+
+
+export function isTCGAPubStudy(studyId:string){
```

Comment:
maybe these should be in ResultsViewPageStoreUtils with isExpressionProfile (name tbd), not that it matters that much though